app/src/main/java/com/android/streetworkapp/ui/profile/AddFriend.kt

```diff
@@ -39,16 +40,23 @@ import com.android.sample.R
 import com.android.streetworkapp.device.bluetooth.BluetoothClient
 import com.android.streetworkapp.device.bluetooth.BluetoothConstants
 import com.android.streetworkapp.device.bluetooth.BluetoothServer
+import com.android.streetworkapp.model.progression.ScoreIncrease
 import com.android.streetworkapp.model.user.UserViewModel
+import com.android.streetworkapp.ui.navigation.NavigationActions
+import com.android.streetworkapp.ui.progress.updateAndDisplayPoints
 import com.android.streetworkapp.ui.theme.ColorPalette.INTERACTION_COLOR_DARK
 import com.android.streetworkapp.ui.theme.ColorPalette.PRIMARY_TEXT_COLOR
 import com.android.streetworkapp.ui.theme.ColorPalette.PRINCIPLE_BACKGROUND_COLOR
 import com.android.streetworkapp.ui.theme.ColorPalette.SECONDARY_TEXT_COLOR
 import com.android.streetworkapp.ui.theme.LightGray
+import kotlinx.coroutines.CoroutineScope
 
 @Composable
 fun AddFriendScreen(
     userViewModel: UserViewModel,
+    navigationActions: NavigationActions? = null,
```

Comment:
Why is it a nullable var? navigationActions will always be defined

Reply:
You are right, this is updated thank you!